lib/rake/build.rake

```diff
@@ -119,6 +119,8 @@ namespace :build do
         # Start new workers
         if rack_env?(:production)
           RakeUtils.system 'bin/delayed_job', '-n', '50', 'start'
+        elsif rack_env?(:test)
```

Comment:
It's a bit more precise to use `CDO.test_system?` because `rack_env` identifies what type of system are we executing in. `rack_env?(:test)` is true for a unit test executing on an engineer's local development environment, and any code executing in a continuous integration build. Those environments don't run our `aws/ci_build` process so they'll never execute this code path, but might be good to be more specific.

Reply:
Aha! That's is exactly what I'd been searching for, I was very uncomfortable using :test, TY!